### PR TITLE
Api controller

### LINKED ISSUE #44: Rest-API
The API which will be used to test our program. (This will be split into multiple tasks)

### PR DESCRIPTION
**Pull request by Hjaltesorgenfrei**
Api controller to the spec. passes all tests in the Api spec folder. 
fixes #44

### CODE REVIEW

File: MiniTwit.Models/UserRepository.cs
Comment (jlndk) [inline MiniTwit.Models/UserRepository.cs]: Can this be solved with dependency injection?

File: MiniTwit.Entities/Follows.cs
Comment (jlndk) [inline MiniTwit.Entities/Follows.cs]: Isn't the convension to name Entities/Models in singular?

File: MiniTwit.Web.App/Controllers/ApiController.cs
Comment (jlndk) [inline MiniTwit.Web.App/Controllers/ApiController.cs]: Warning fra CI

File: MiniTwit.Web.App/Controllers/ApiController.cs
Comment (jlndk) [inline MiniTwit.Web.App/Controllers/ApiController.cs:201]: Would like to have these TODO's solved

**Review by jlndk — CHANGES_REQUESTED** [review]
Besides the minor points I've found, this PR should wait until #67 is merged.

File: MiniTwit.Web.App/Controllers/ApiController.cs
Comment (Hjaltesorgenfrei) [reply MiniTwit.Web.App/Controllers/ApiController.cs:201]: Nope its supposed to be there.

**Review by Duckapple — APPROVED** [review]
Seems cool enough, and I trust Hjalte

**Review by jlndk — APPROVED** [review]
I'll trust you guys

**Pull request merged**
